inc/template.php

```diff
@@ -676,33 +679,39 @@ function tpl_searchform($ajax = true, $autocomplete = true) {
  * @author Andreas Gohr <andi@splitbrain.org>
  *
  * @param string $sep Separator between entries
- * @return bool
+ * @param bool   $return return or print
+ * @return bool|string
  */
-function tpl_breadcrumbs($sep = '•') {
+function tpl_breadcrumbs($sep = null, $return = false) {
     global $lang;
     global $conf;
 
     //check if enabled
     if(!$conf['breadcrumbs']) return false;
 
+    //set default
+    if(is_null($sep)) $sep = '•';
```

Comment:
Why the change from the default value in the signature to implementing it in code here?

Reply:
Yeah, but isn't that exactly the behavior if the default value is set in the signature?

Reply:
Yes, you are right! Sorry for reading it only superficially :/